charts/tempo/templates/service.yaml

```diff
@@ -35,10 +35,16 @@ spec:
   - name: tempo-prom-metrics
     port: 3100
     targetPort: 3100
+    {{- with .Values.serviceMonitor.appProtocol }}
+    appProtocol: {{ . }}
```

Comment:
afaik there's no way in Tempo to configure this to be anything other than http. does it make sense to just hardcode these to "http"?

this may be a mild breaking change. does it change a service or port name in k8s?